site/generate-jenkins-io-template.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+WWW_ROOT_DIR="${WWW_ROOT_DIR:-${1:-./www2}}"
+SELF_URL="${SELF_URL:-${2:-https://updates.jenkins.io}}"
```

Comment:
Please add a comment to the beginning of the script that explains usage.

Reply:
Thanks for adding the doc, but by usage I meant something like https://github.com/jenkins-infra/update-center2/blob/c5841e67ed066988bf4c3ce671fc2c1c95b3382a/site/generate.sh#L3

(Since the other script cannot be used without args it's part of the default invocation, that cannot be done here of source. Check for `$# -gt 2` might still be a useful error condition.